repos/c2sm/packages/cosmo-eccodes-definitions/package.py

```diff
@@ -32,6 +32,7 @@ class CosmoEccodesDefinitions(Package):
 
     maintainers = ['egermann']
```

Comment:
Should `maintainers` be updated as well here?

Reply:
Good point! I think we should discuss this for all packages, that still contain @elsagermann, in one of our team meeting.